components/support/sync15-traits/src/payload.rs

```diff
@@ -40,6 +40,52 @@ impl Payload {
         self
     }
 
+    /// "Auto" fields are fields like 'sortindex' (and potentially 'ttl' in
+    /// the future) which are:
+    ///
+    /// - Added to the payload automatically when deserializing if present on
+    ///   the incoming BSO.
+    /// - Removed from the payload automatically and attached to the BSO if
+    ///   present on the outgoing payload.
```

Comment:
```suggestion
    ///   the incoming BSO or envelope.
    /// - Removed from the payload automatically and attached to the BSO or
    ///   envelope if present on the outgoing payload.
```

...Since the envelope takes the place of the BSO on Desktop.